packages/cactus-cmd-api-server/src/main/typescript/config/config-service.ts

```diff
@@ -587,24 +587,22 @@ export class ConfigService {
     env?: NodeJS.ProcessEnv;
     args?: string[];
   }): Config<ICactusApiServerOptions> {
-    if (!ConfigService.config) {
```

Comment:
@petermetz Can be #720 solved by this way passing the `overrides` parameter from `newExampleConfigConvict` to `getOrCreate` and letting this function call `create` when appropriate?

Reply:
@elenaizaguirre I think they are different because #720 is about completely deleting certain unused configuration variables from the code of the API server (variables that turned out to be better matched to specific plugins not the API server itself)